src/omv/modules/py_tf.c

```diff
@@ -120,11 +120,84 @@ STATIC mp_obj_t py_tf_model_output_subscr(mp_obj_t self_in, mp_obj_t index, mp_o
     return MP_OBJ_NULL; // op not supported
 }
 
+STATIC mp_obj_t py_tf_model_output_get_image(uint n_args, const mp_obj_t *pos_args, mp_map_t *kw_args) {
```

Comment:
This should be a generic image module function, in py_image.c, passed the raw output, scale etc..

Reply:
We should extend Image() to accept a list[] so you can create an image from that. Or, have like a image.from_list() function in the image module.

Reply:
@kwagyeman Could implement that change ? I want to test the output callback more and that would make it a lot easier for me to do so. But please keep changes to py_tf to absolute minimum as I already have stashed changes for that file.

Reply:
Yeah, it would be in py_image.c so, no changes to py_tf.